x-pack/elastic-agent/magefile.go

```diff
@@ -58,7 +58,9 @@ func init() {
 
 	devtools.BeatDescription = "Agent manages other beats based on configuration provided."
 	devtools.BeatLicense = "Elastic License"
-	devtools.Platforms = devtools.NewPlatformList("!linux/386 !windows/386")
+
+	devtools.Platforms.Filter("linux/386")
+	devtools.Platforms.Filter("windows/386")
```

Comment:
Does the `filter` function returns an immutable list that should be re-assigned? Or modifies the existing object?

```suggestion
	devtools.Platforms = devtools.Platforms.Filter("linux/386")
	devtools.Platforms = devtools.Platforms.Filter("windows/386")
```